gatsby-config.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env node */
+
+const path = require('path')
+
+require('./config/prismjs/dvc')
+require('./config/prismjs/usage')
+
+const apiMiddleware = require('./middleware/api')
+const redirectsMiddleware = require('./middleware/redirects')
+
+const title = 'Data Version Control · DVC'
+const description =
+  'Open-source version control system for Data Science and Machine Learning ' +
+  'projects. Git-like experience to organize your data, models, and ' +
+  'experiments.'
+
+const keywords = [
+  'data version control',
+  'machine learning',
+  'models management'
+]
+
+const plugins = [
+  {
+    resolve: 'gatsby-source-filesystem',
+    options: {
+      name: 'blog',
+      path: path.join(__dirname, 'content', 'docs')
+    }
+  },
+  {
+    resolve: 'gatsby-transformer-remark',
+    options: {
+      plugins: [
+        'gatsby-remark-dvc-linker',
+        'gatsby-remark-prismjs',
+        'gatsby-remark-copy-linked-files',
+        'gatsby-remark-smartypants',
+        {
+          resolve: 'gatsby-remark-external-links'
+        },
+        {
+          resolve: 'gatsby-remark-autolink-headers',
+          options: {
+            enableCustomId: true,
+            isIconAfterHeader: true
+          }
+        }
+      ]
+    }
+  },
+  'gatsby-plugin-catch-links',
+  /*{
```

Comment:
remove commented code

Reply:
Uncommented it actually, and added favicon, totally forgot about it.